### ISSUE TITLE
stealremotepwds - different salts for same user?

### ISSUE THREAD

**Issue by Meatballs1** [quentinhardy/odat#11]
If I use --test-module I get the same salt each time for a specific user (I already know password)

If I use --get-all-passwords I get a different salt than for the test-module.

If I try decrypt-sessions on the known user it wont decrypt the values? :(

**Comment by Meatballs1** [comment]
Looks like its not parsing user list very well? 

```
09:49:02 INFO -: Getting remote passwords of 2 users
09:49:02 INFO -: Try to get the session key and salt of the XXXX user
09:49:02 DEBUG -: Session key and salt are now emply
09:49:02 DEBUG -: Sniffing is running in a new thread
09:49:02 DEBUG -: Session key and salt are now emply
09:49:02 DEBUG -: Waiting 3 seconds
09:49:05 DEBUG -: Connection to the database via a new thread with the username None
09:49:05 DEBUG -: Connecting with None/aaaaaaa@
09:49:05 INFO -: We have captured the session key: 
09:49:05 INFO -: We have captured the salt: 
```

**Comment by Meatballs1** [comment]
Maybe not being passed correctly `b = Thread(None, self.__try_to_connect__, None, (), {'args':self.args})` ?

**Comment by Meatballs1** [comment]
Ah:

```
  def __try_to_connect__(self, user):
                '''
                Establish a connection to the database
                '''
                import cx_Oracle
                try:
                        connectString = "{0}/{1}@{2}:{3}/{4}".format(user, 
```

Its trying to use `self.args['user']` instead of the user supplied one...

**Comment by quentinhardy** [comment]
Thank you Meatballs1. Bug fixed in last commit (493035261bf44f373849f146c8a8ccad6cf85b1c).

**Comment by Meatballs1** [comment]
Awesome thanks :)
